README.md

```diff
@@ -24,17 +23,51 @@ This Kubernetes-based version is built using the same approach as the [machine c
 a situation where your Kubernetes cluster and its ingress controllers have
 a fast connection to the Internet.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [Content Cache Documentation](https://charmhub.io/content-cache-k8s/docs).
+
+## Get Started
+
+To begin, refer to the [Getting Started](https://charmhub.io/content-cache-k8s/docs/tutorial-getting-started) tutorial for step-by-step instructions.
+
+### Basic Operations
+
+The following actions are available for the charm:
+- report-visits-by-ip
+
+Tuning options include:
+- cache storage size
+- maximum request size to cache 
+- cache validity duration
+
+You can find more information about supported actions in [the Charmhub documentation](https://charmhub.io/content-cache-k8s/actions).
+
+## Integrations
+
+Since Content-cache is meant to serve as cache for another charm, you can use Hello-kubecon as an example:
```

Comment:
```suggestion
Since Content-cache is meant to serve as cache for another charm. You can use [hello-kubecon](https://charmhub.io/hello-kubecon) as an example:
```

Reply:
Removing the "since" in the beginning as well.